CIP-0072/README.md

```diff
@@ -76,14 +72,14 @@ The on chain dApp registration certificate MUST follow canonical JSON and be ser
 ```
 
 ### Properties
-*`subject`*: Identifier of the claim subject (dApp). A UTF-8 encoded string. This uniqueness of this property cannot be guaranteed by the protocol and multiple claims for the same subject may exist, therefore it is required to exist some mechanism to assert trust in the *veracity* of this property.
+*`subject`*: Identifier of the claim subject (dApp). A UTF-8 encoded string, max 64 chars. The uniqueness of this property cannot be guaranteed by the protocol and multiple claims for the same subject may exist, therefore it is required to exist some mechanism to assert trust in the *veracity* of this property.
```

Comment:
If we put 64 chars (which is ok), then we should also update JSON schema to reflect this.

Reply:
Yep, just added this line everywhere:
1. Properties section - subject's description
2. On-chain JSON Schema
3. Off-chain JSON Schema